packages/backend/src/modules/activity2/indexers/DayActivityIndexer.ts

```diff
@@ -0,0 +1,36 @@
+import { ManagedChildIndexer } from '../../../tools/uif/ManagedChildIndexer'
+import { DayActivityIndexerDeps } from './types'
+
+export class DayActivityIndexer extends ManagedChildIndexer {
+  constructor(private readonly $: DayActivityIndexerDeps) {
+    const logger = $.logger.tag($.projectId)
+    super({ ...$, logger, name: `activity_indexer_${$.projectId}` })
+  }
+
+  override async update(from: number, to: number): Promise<number> {
+    // TODO: confirm if this logic is still needed
```

Comment:
have you confirmed?

Reply:
We need more time/experiments to confirm this